R/mht01.R

```diff
@@ -105,14 +105,14 @@ mht01_pre <- function(adam_db, ...) {
 #' @returns the postprocessing function returns an `rtables` object or an `ElementaryTable` (null report).
 #' @export
 #'
-mht01_post <- function(tlg, prune_0 = TRUE, ...) {
+mht01_post <- function(tlg, row_split_var = "MHBODSYS", prune_0 = TRUE, ...) {
   if (prune_0) {
     tlg <- smart_prune(tlg)
   }
 
   tbl_sorted <- tlg %>%
     sort_at_path(
-      path = c("MHBODSYS", "*", "MHDECOD"),
+      path = c(row_split_var, "*", "MHDECOD"),
```

Comment:
if there are multiple `row_split_var`, will this still work?

Reply:
good point, now it does.